### PR TITLE
Feature/dao dashboard

### PR DESCRIPTION
**Pull request by VERZUOL1**

### CODE REVIEW

File: components/AreaChartRenderer/components/range-toggle/RangeToggle.tsx [hunk 1]
```diff
@@ -33,23 +35,48 @@ const TOGGLE_SET = [
...
+          onChange={v => {
+            onClick(v as Range);
+          }}
+          options={TOGGLE_SET.map(({ label, type }) => ({
+            label,
+            value: type,
+          }))}
+        />
+      ) : (
+        <>
```
Comment (AlexeyPonomarenko) [inline components/AreaChartRenderer/components/range-toggle/RangeToggle.tsx]: I consder <></> is redundant in this case)

File: components/AreaChartRenderer/components/range-toggle/RangeToggle.tsx [hunk 2]
```diff
@@ -33,23 +35,48 @@ const TOGGLE_SET = [
...
-      </Button>
-    ))}
-  </div>
-);
+const RangeToggle: FC<RangeToggleProps> = ({
+  onClick,
+  activeRange,
+  className,
+}) => {
+  const isMobile = useMedia('(max-width: 768px)');
```
Comment (AlexeyPonomarenko) [inline components/AreaChartRenderer/components/range-toggle/RangeToggle.tsx]: I would suggest to astro mobile 920 px. What do you think?)

File: components/AreaChartRenderer/components/range-toggle/RangeToggle.module.scss [hunk 3]
```diff
@@ -12,6 +12,15 @@
   }
 }
 
+.select {
+  p {
```
Comment (AlexeyPonomarenko) [inline components/AreaChartRenderer/components/range-toggle/RangeToggle.module.scss]: maybe we can use classes here?)

Comment (VERZUOL1) [reply components/AreaChartRenderer/components/range-toggle/RangeToggle.module.scss]: fixed

File: components/AreaChartRenderer/components/range-toggle/RangeToggle.tsx (hunk 2 above)
Comment (VERZUOL1) [reply components/AreaChartRenderer/components/range-toggle/RangeToggle.tsx]: fixed

**Review by AlexeyPonomarenko — APPROVED** [review]

**Pull request merged**